pr-lint/src/index.js

```diff
@@ -17,15 +17,15 @@ async function main() {
   }
 
   const { title } = contextPullRequest;
-  if (title === "Update index.html") {
-    // This is the default title you get when you use the GitHub UI to
+  if ((title === "Update index.html") || (title === "Update index.md")) {
```

Comment:
```suggestion
  if (title === "Update index.html" || title === "Update index.md") {
```

I'm not sure why `prettier` didn't complain about the excess brackets. Hmm..